helper/schema/shims_test.go

```diff
@@ -147,7 +147,7 @@ func TestShimResourceApply_create(t *testing.T) {
 		},
 	}
 
-	actual, err := r.Apply(s, d, nil)
+	actual, err := r.Apply(nil, s, d, nil)
```

Comment:
can use `context.Background` here (or `context.TODO` depending on the scenario)

Reply:
Ah yes, forgot about this file, I meant to use `context.Background()`